### PR TITLE
Registration page

### PR DESCRIPTION
**Pull request by vikawal**
Here's our team work about registration page and new input changes, Please, value it)

### CODE REVIEW

File: client/components/input/index.mjs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { inputListener } from "./input.mjs";
```
Comment (TatianaBortnik) [inline client/components/input/index.mjs:1]: Proposition: to name this file example.js or something like

**Review by TatianaBortnik — APPROVED** [review]

File: client/components/Registration-Page/register.css [hunk 2]
```diff
@@ -0,0 +1,59 @@
+@import '../typography/typography.css';
```
Comment (tbazko) [inline client/components/Registration-Page/register.css:1]: to keep our naming consistent, switch folder name to camelCase. Meaning `registrationPage`

File: client/components.html [hunk 3]
```diff
@@ -1,16 +1,25 @@
...
-    <li><a href="./components/input/input">Input</a></li>
+  <h2 class="heading-h2">Components</h2>
+  <ul class="heading-h5">
     <li><a href="./components/typography/typography.html">Typography</a></li>
+    <li><a href="./components/button/button.html">Button</a></li>
+    <li><a href="./components/input/input.html">Input</a></li>
+  </ul>
+  <h2 class="heading-h2">Forms</h2>
+  <ul class="heading-h5">
+    <li><a href="./components/Registration-Page/register.html">Registration page</a></li>
```
Comment (tbazko) [inline client/components.html:21]: If this is complete page, no need to put it in components. Just put it at the same level as index.html
Then, if you keep it's name `register.html`, after you start server you can access it by path `/register`

File: client/components/Registration-Page/register.mjs [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+function checkPasswords(item) {
+   const passwordVarification = document.getElementById('password2');
+   (item.value != '') ? passwordVarification.setAttribute('pattern', item.value) : passwordVarification.setAttribute('pattern', '');
+}
+
+inputListener('inputCheckbox', 'input', showPassword);
+function showPassword() {
+   const inputPasswordFirst = document.getElementById("password1");
+   const inputPasswordSecond = document.getElementById("password2");
+   const Status = (input) => (input.type === 'password') ? 'text' : 'password';
```
Comment (tbazko) [inline client/components/Registration-Page/register.mjs:27]: It's a function. Functions are typically written with small letter and name is usually a verb. So I would rename it to `getStatus`

File: client/components/Registration-Page/register.mjs [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+      }
+      if (!item.checkValidity()) {
+         validate = false;
+      }
+   }
+   (validate) ? document.getElementById('createAccount').disabled = false : document.getElementById('createAccount').disabled = true;
+}
+
+function checkPasswords(item) {
+   const passwordVarification = document.getElementById('password2');
```
Comment (tbazko) [inline client/components/Registration-Page/register.mjs:19]: Small nitpick, it's `passwordVerification`

**Review by tbazko — CHANGES_REQUESTED** [review]

**Pull request merged**